install_config/install/advanced_install.adoc

```diff
@@ -626,6 +626,63 @@ openshift_master_named_certificates=[{"certfile": "/root/STAR.openshift.com.crt"
 openshift_master_overwrite_named_certificates=true
 ----
 
+[[advanced-install-deployment-types]]
+=== Configuring Deployment Type
+
+Various defaults used throughout the playbooks and roles in this repository are
+set based on the deployment type configuration (usually defined in an Ansible
+hosts file).
+
+ifdef::openshift-enterprise[]
+Ensure the `openshift_deployment_type` parameter in your inventory file is set to
+`openshift-enterprise`.
+endif::[]
+ifdef::openshift-origin[]
+Ensure the `openshift_deployment_type` parameter in your inventory file is set to
+`origin`.
+endif::[]
+
+.Defaults for Deployment Type
+[options="header"]
+|===
+
+|Deployment Type |Default Variable
+
+|`openshift.common.service_type`
+(also used for package names)
+|
+ifdef::openshift-enterprise[]
+`atomic-openshift`
+endif::[]
+ifdef::openshift-origin[]
+`origin`
+endif::[]
+
+|`openshift.common.config_base`
+|*_/etc/origin_*
+
+|`openshift.common.data_dir`
+|`/var/lib/origin`
+
+|`openshift.master.registry_url`, `openshift.node.registry_url`
+|
+ifdef::openshift-enterprise[]
+`openshift3/ose-${component}:${version}`
+endif::[]
+ifdef::openshift-origin[]
+`openshift/origin-${component}:${version}`
+endif::[]
+
+|Image Streams
+|
+ifdef::openshift-enterprise[]
+`rhel`
+endif::[]
+ifdef::openshift-origin[]
+`centos`
+endif::[]
+|===
+
```

Comment:
That's what I meant with my [previous comment](https://github.com/openshift/openshift-docs/pull/4208#discussion_r112932287), though I wrote it in the header of this section :-)

I feel those are internal details to the installer code base. They are documented in:
https://github.com/openshift/openshift-ansible/blob/master/DEPLOYMENT_TYPES.md

And the only reason I see for it being there is to assist people developing Ansible roles / code to understand what effect `[openshift_]deployment_type` has on other variables.